src/lib/constants.js

```diff
@@ -43,3 +43,73 @@ export const defaultColors = {
     l: 65
   },
 };
+
+export const presetColors = [
```

Comment:
for now i just added two preset colors to demonstrate the feature. We might want to move this to another file since it could get really long.